pagopa/pn-frontend
# fix(PN-10623): implement field for senderDenomination

## Short description
Implemented field for senderDenomination with check on length

## List of changes proposed in this pull request
- Added field `senderDenomination` (composed by organization.parentDescription +  organization.name) in preliminary informations page and made it required

## How to test
Login as PA and go to newNotification page. Check field "Ente mittente" is present and has maximum 80 characters.
